Tests/Integration/CloudToDeviceMessageSizeLimitShouldAcceptTests.cs

```diff
@@ -141,7 +141,7 @@ public class CloudToDeviceMessageSizeLimitShouldAcceptTests : CloudToDeviceMessa
             if (expectedMacCommandsCount > 0)
             {
                 // Possible problem: Manually casting payloadDataDown.Fopts to array and reversing it
```

Comment:
Remove the comment

Reply:
Done with 4b057e1b4542dcc991592714f6fefe59a6577295.